mmv1/products/networkservices/HttpRoute.yaml

```diff
@@ -0,0 +1,507 @@
+                  name: pathRedirect
+                  description: |
+                    The path that will be used in the redirect response instead of the one that was supplied in the request.
+                - !ruby/object:Api::Type::String
+                  name: prefixRewrite
+                  description: |
+                    Indicates that during redirection, the matched prefix (or path) should be swapped with this value.
+                - !ruby/object:Api::Type::String
+                  name: responseCode
+                  description: |
+                    The HTTP Status code to use for the redirect.
+                - !ruby/object:Api::Type::Boolean
+                  name: httpsRedirect
+                  description: |
+                    If set to true, the URL scheme in the redirected request is set to https.
+                - !ruby/object:Api::Type::Boolean
+                  name: stripQuery
+                  description: |
+                    If set to true, any accompanying query portion of the original URL is removed prior to redirecting the request. 
+                - !ruby/object:Api::Type::Integer
+                  name: portRedirect
+                  description: |
+                    The port that will be used in the redirected request instead of the one that was supplied in the request.
+            - !ruby/object:Api::Type::NestedObject
+              name: faultInjectionPolicy
+              description: |
+                The specification for fault injection introduced into traffic to test the resiliency of clients to backend service failure.
+              properties:
+                - !ruby/object:Api::Type::NestedObject
+                  name: delay
+                  description: |
+                    Specification of how client requests are delayed as part of fault injection before being sent to a destination.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: fixedDelay
+                      description: |
+                       Specify a fixed delay before forwarding the request.
+                    - !ruby/object:Api::Type::Integer
+                      name: percentage
+                      description: |
+                         The percentage of traffic on which delay will be injected.
+                - !ruby/object:Api::Type::NestedObject
+                  name: abort
+                  description: |
+                     Specification of how client requests are aborted as part of fault injection before being sent to a destination.
+                  properties:
+                    - !ruby/object:Api::Type::Integer
+                      name: httpStatus
+                      description: |
+                         The HTTP status code used to abort the request.
+                    - !ruby/object:Api::Type::Integer
+                      name: percentage
+                      description: |
```

Comment:
Add send_empty_value: true to allow 0 is set.

Reply:
Done